pkgs/by-name/te/tenv/package.nix

```diff
@@ -0,0 +1,27 @@
+{ buildGoModule, lib, fetchFromGitHub }:
+buildGoModule rec {
+  pname = "tenv";
+  version = "v1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "tofuutils";
+    repo = "tenv";
+    rev = version;
+    sha256 = "sha256-yLNdBwKF6Jts661P+YZhFGNr71TG7Scb6RGvFxTLqYQ=";
+  };
+
+  vendorHash = "sha256-GGWiP1rIDF6qxST2ZmnKNkgbS+15hxaCs1d1+UEiYgU=";
+
+  # Disable tests since it requires network access and relies on the
+  # presence of release.hashicorp.com
+  doCheck = false;
+
+  meta = with lib; {
+    description =
+      "OpenTofu, Terraform and Terragrunt version manager, written in Go.";
+    mainProgram = "tenv";
+    homepage = "https://github.com/tofuutils/tenv";
+    license = licenses.mit;
```

Comment:
Looking at the Repo, it uses the Apache License (https://github.com/tofuutils/tenv/blob/main/LICENSE), not the MIT.